site/en/guide/tensor.ipynb

```diff
@@ -148,7 +148,7 @@
         "id": "G3IJG-ug_H4u"
       },
       "source": [
-        "A \"matrix\" or \"rank-2\" tensor has 2-axes:"
```

Comment:
This should actually be "two axes". According to the Google style guide, we spell out numbers under 10: https://developers.google.com/style/numbers

And for tensors, according to the TensorFlow style guide we use something like "two-dimensional tensor with shape [...]": https://www.tensorflow.org/community/contribute/docs_style#tensors

Can you make those changes while you're in here? Thanks

Reply:
Sure, will do that. Thanks for the guidance.